### PR TITLE
i18n(es): update `/6-islands/1.mdx` & `/5-astro-api/3.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `/6-islands/1.mdx` & `/5-astro-api/3.mdx` based on this commit 9ec3861c8f24aaeedebeb08e5f018e7213146faf

Could you please kindly add `hacktober-accepted` tag?

### CODE REVIEW

File: src/content/docs/es/tutorial/6-islands/1.mdx [hunk 1]
```diff
@@ -97,7 +97,7 @@ Este componente tomará un array de mensajes de bienvenida como props y seleccio
     </BaseLayout>
     ```
 
-  5. Vuelve a visitar tu página y compara los dos componentes. El segundo botón funciona porque la directiva `client:load` le dice a Astro que envíe y vuelva a ejecutar su JavaScript al _client_ cuando la página _loads_, haciendo que el componente sea interactivo. Esto se llama un componente **hidratado**.
+  5. Vuelve a visitar tu página y compara los dos componentes. El segundo botón funciona porque la directiva `client:load` le dice a Astro que envíe y vuelva a ejecutar su JavaScript en el _client_ cuando la página _loads_, haciendo que el componente sea interactivo. Esto se llama un componente **hidratado**.
```
Comment (Waxer59) [inline src/content/docs/es/tutorial/6-islands/1.mdx:100]:
```suggestion
  5. Vuelve a visitar tu página y compara los dos componentes. El segundo botón funciona porque la directiva `client:load` le dice a Astro que envíe y vuelva a ejecutar su JavaScript en el _cliente_ cuando la página _cargue_, haciendo que el componente sea interactivo. Esto se llama un componente **hidratado**.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**